foxpuppet/windows/browser.py

```diff
@@ -2,43 +2,59 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from foxpuppet.windows import Windows
-
 from selenium.webdriver.common.by import By
 
+from foxpuppet.expected import new_browser_window_is_opened
+from foxpuppet.windows.base import BaseWindow
```

Comment:
We should import `BaseWindow` from the `windows` module. I also notice that we switch between relative and absolute imports. We can address this in a separate patch though.